docs/HyperIndex/contract-import.md

```diff
@@ -1,28 +1,26 @@
 ---
 id: contract-import
-title: No-Code Quickstart
-sidebar_label: No-Code Quickstart
+title: Quickstart
+sidebar_label: Quickstart
 slug: /contract-import
 ---
 
-The Envio CLI allows you to quickly autogenerate a basic indexer. This is the quickest way to get going and the likely starting point for most developers.
+The Quickstart allows you to instantly autogenerate a basic indexer and query your data. This is the quickest way to get going and likely the starting point for most developers.
 
-For example, you could autogenerate an indexer for Eigenlayer AND index the entire Eigenlayer contract in under 5 minutes simply through running `envio init` and pasting the contract address: https://etherscan.io/address/0x858646372cc42e1a627fce94aa7a7033e7cf075a
+For example, you could autogenerate an indexer for Eigenlayer AND index the entire Eigenlayer contract in under 5 minutes simply through running `envio init` and pasting the contract address from the Block Explorer: https://etherscan.io/address/0x858646372cc42e1a627fce94aa7a7033e7cf075a
 
-You can also autogenerate an indexer with a local ABI.
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zkVlGgf5XAo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JOiLUysZf-s" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Instructions
 
 To start, simply run:
 
 ```
 envio init
 ```
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/zkVlGgf5XAo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
-<iframe width="560" height="315" src="https://www.youtube.com/embed/JOiLUysZf-s" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Don't want this video elsewhere maybe?

Reply:
i didn't remove the videos, they still there, just cut those lines and pasted into a different position on the same page.